pod/live/views.py

```diff
@@ -315,13 +315,22 @@ def render_event_template(request, evemnt, user_owns_event):
 
 def events(request):  # affichage des events
 
+    # Todo reprendre les filtres pour ne pas afficher certains évènements
+    # if request.user.is_authenticated:
+    #     access_group_filter = Q(restrict_access_to_groups__in=[
+    #         group_id for group_id in request.user.owner.accessgroups.values_list("id")
+    #     ])
+    #     restricted_filter = Q()
+    # else:
+    #     access_group_filter = Q()
+    #     restricted_filter = Q(is_restricted=False)
+
     queryset = Event.objects.filter(
-        Q(start_date__gt=datetime.now()) & Q(end_date__gte=datetime.now())
+        Q(end_date__gt=timezone.now())
+        & Q(is_draft=False)
```

Comment:
pourquoi Q() ? ne pourrait-on pas mettre queryset = Event.objects.filter(end_date__gt=timezone.now(), is_draft=False) ? qu'est-ce que Q apporte ? Merci

Reply:
c'était la seule manière de pour pouvoir ajouter un filtre 'or' au milieu de plusieurs 'and' .
Là c'est vrai que ça ne sert plus (ou peut être qu'on peut faire différemment)

Reply:
oui ok pour utiliser Q pour des Or mais pour des simples and, on peut le retirer, merci !